tests/informatics/test_constraints.py

```diff
@@ -18,6 +18,17 @@ def scalar_range_constraint() -> ScalarRangeConstraint:
     )
 
 
+@pytest.fixture
```

Comment:
No need to define this as a fixture if it's only used in one test

Reply:
I'm following the pattern used by all other tests in this file.

Reply:
It's a bad pattern because it splits the test logic into two places without providing any reproducibility benefits.

Looking at the history, when this file was first created each fixture was used for multiple tests. But that's no longer the case (except for the ingredient ratio constraint)

Reply:
Sure, its probably a bad pattern. But i'd rather maintain style consistency with the rest of the file